.github/workflows/bleeding-edge.yaml

```diff
@@ -40,16 +40,19 @@ jobs:
         # the '-dev' suffix allows to use pre-releases if no final release is available yet
         # this version should be upgraded as often as possible, typically once a year when
         # Cython, numpy and matplotlib are known to be compatible
-        python-version: '3.12-dev'
+        python-version: '3.13-dev'
 
     - name: Install dependencies
+      # pre-installing kiwisolver (mpl dependency) because it doesn't have wheels
```

Comment:
looks like kiwisolver actually released since you wrote this and now has wheels available (https://pypi.org/project/kiwisolver/1.4.7/).

Reply:
indeed. Let me remove this bit as I rebase the branch